packages/ecc-client-ga4gh-tes/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "ecc-client-ga4gh-tes",
+  "description": "Web componets for TES",
+  "main": "index.js",
```

Comment:
Is output location really this?

Reply:
Yes! after compilation all the modules/components can be imported from src/index.js of build/out-tsc folder, is there something wrong with that :thinking: or did I miss something?

Reply:
done